Woof/Woof/View/ReusableViews/FiveStarRatingView.swift

```diff
@@ -0,0 +1,39 @@
+import SwiftUI
+
+/// A view displaying a rating of the person.
+struct FiveStarRatingView: View {
+    /// The rating of the person.
+    let rating: Int
+
+    /// The text will be displayed if the rating is not specified.
+    private let noRatingText = "not rated"
```

Comment:
Please do not mix private and non-private interfaces.

Reply:
Fixed it.